src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -600,7 +600,7 @@ async Task<bool> IsUpToDateInternalAsync(CancellationToken token)
                 finally
                 {
                     LastCheckTimeUtc = DateTime.UtcNow;
-                    logger.Verbose("Up to date check completed in {0:#,##0.#} ms", sw.Elapsed.TotalMilliseconds);
```

Comment:
Not that your change here is bad, but the old code is culture aware too (assuming logger.Verbose doesn't do something whacky and custom under the covers)...
![image](https://user-images.githubusercontent.com/754264/62330778-04b1d680-b4fd-11e9-9068-c6579157f82d.png)

Reply:
TIL!